webview_panels/src/lib/altimate/types.js

```
@@ -0,0 +1,663 @@
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+var Oe;
+function vr() {
+  if (Oe) return F;
+  Oe = 1;
+  var u = Ce, j = Symbol.for("react.element"), V = Symbol.for("react.fragment"), T = Object.prototype.hasOwnProperty, L = u.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactCurrentOwner, W = { key: !0, ref: !0, __self: !0, __source: !0 };
+  function x(h, d, O) {
+    var g, _ = {}, m = null, U = null;
+    O !== void 0 && (m = "" + O), d.key !== void 0 && (m = "" + d.key), d.ref !== void 0 && (U = d.ref);
+    for (g in d) T.call(d, g) && !W.hasOwnProperty(g) && (_[g] = d[g]);
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)